client/my-sites/people/delete-user/index.jsx

```diff
@@ -295,11 +317,22 @@ class DeleteUser extends React.Component {
 	}
 }
 
+const getContributorType = ( externalContributors, userId ) => {
+	if ( externalContributors.data ) {
+		return externalContributors.data.includes( userId ) ? 'external' : 'standard,';
```

Comment:
it doesn't look like this `,` was expected.

how do you feel about moving to `'external' | 'standard' | 'pending'`?

note that if we made a type for this we'd get autocomplete; don't block this PR though on adding types to this file.